pinot-core/src/main/java/org/apache/pinot/core/geospatial/transform/function/ScalarFunctions.java

```diff
@@ -188,4 +188,40 @@ public static double stDistance(byte[] firstPoint, byte[] secondPoint) {
       return firstGeometry.isEmpty() || secondGeometry.isEmpty() ? Double.NaN : firstGeometry.distance(secondGeometry);
     }
   }
+
+  @ScalarFunction(names = {"stContains", "st_contains"})
+  public static int stContains(byte[] first, byte[] second) {
+    Geometry firstGeometry = GeometrySerializer.deserialize(first);
+    Geometry secondGeometry = GeometrySerializer.deserialize(second);
+    if (GeometryUtils.isGeography(firstGeometry) != GeometryUtils.isGeography(secondGeometry)) {
+      throw new RuntimeException("The first and second arguments should either both be geometry or both be geography");
+    }
+    // TODO: to fully support Geography contains operation.
```

Comment:
See https://github.com/apache/pinot/pull/8620#issuecomment-1118872760

Reply:
What is the relation between the TODO and the linked GH comment? Anyway, if you think it is important enough to add it here as a comment in the PR, I guess you should also add something like`See #8620` in the comment in the code

Reply:
The same TODO was added to the corresponding transform functions in https://github.com/apache/pinot/pull/8620/ and the explanation is in the linked GH comment. Although this is present in the [public documentation](https://docs.pinot.apache.org/basics/indexing/geospatial-support#relationship) as well so I can remove the TODO comment here if you'd like (since the original PR is over 2 years ago at this point so not sure if this is important enough to ever be implemented).